packages/js/components/src/tooltip/tooltip.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button, Popover } from '@wordpress/components';
+import { createElement, Fragment, useState } from '@wordpress/element';
+import { Icon, help } from '@wordpress/icons';
+import { KeyboardEvent } from 'react';
+
+type TooltipProps = {
+	children?: JSX.Element | string;
+	text: JSX.Element | string;
+};
+
+export const Tooltip: React.FC< TooltipProps > = ( {
+	children = <Icon icon={ help } />,
+	text,
+} ) => {
+	const [ isPopoverVisible, setIsPopoverVisible ] = useState( false );
+
+	return (
+		<>
+			<div className="woocommerce-tooltip">
+				<Button
+					className="woocommerce-tooltip__button"
+					onKeyDown={ (
+						event: KeyboardEvent< HTMLButtonElement >
+					) => {
+						if ( event.key !== 'Enter' ) {
+							return;
+						}
+						setIsPopoverVisible( true );
+					} }
+					onClick={ () => setIsPopoverVisible( true ) }
+					label={ __( 'Help', 'woocommerce' ) }
+				>
+					{ children }
+				</Button>
+
+				{ isPopoverVisible && (
+					<Popover
+						focusOnMount="container"
+						position="top center"
```

Comment:
It might be a good idea to make this configurable via the Tooltip props as well. I can see use cases where we'd want to control how the Tooltip positions itself more easily.

Reply:
Added in https://github.com/woocommerce/woocommerce/pull/35024/commits/c55b0626e85a4c9e0a242f08f6c73776f7e85279.